samples/graph-bulk-meetings/csharp/EventMeeting/ClientApp/src/components/FileUpload.jsx

```diff
@@ -0,0 +1,154 @@
+                this.setState({
+                    uploadedFileName: fileName,
+                    isFormInvalid: false
+                });
+                this.renderFile(fileObj)
+            }
+            else {
+                this.setState({
+                    isFormInvalid: true,
+                    uploadedFileName: ""
+                })
+            }
+        }
+    }
+
+    // Handler when user click on create button.
+    onCreateTeamsMeetingsButtonClick = (event) => {
+        microsoftTeams.app.getContext().then(async (context) => {
+            var excelrows = [];
+            for (var i = 1; i < this.state.rows.length; i++) {
+                var obj = {
+                    topicName: this.state.rows[i][0],
+                    trainerName: this.state.rows[i][1],
+                    startdate: this.state.rows[i][2],
+                    enddate: this.state.rows[i][3],
+                    timing: this.state.rows[i][4],
+                    participants: this.state.rows[i][5]
+                }
+                excelrows.push(obj);               
+            }
+
+            var response = await axios.post(`api/meeting/${context.user.id}`, excelrows);
+            if (response.status === 201) {
+                microsoftTeams.dialog.submit("Created successfully!");
+            }
+        });
+    }
+
+    toggle() {
+        this.setState({
+            isOpen: !this.state.isOpen
+        });
+    }
+
+    openFileBrowser = () => {
+        this.fileInput.current.click();
+    }
+
+    openNewPage = (chosenItem) => {
+        const url = chosenItem === "github" ? "https://github.com/react-excel-renderer" : "https://medium.com/render-and-display-excel-sheets-on-webpage-using-react-js-af785a5db6a7";
+        window.open(url, '_blank');
+    }
+
+
+
+    render() {
+        return (
+            <div>
+                <div>
+
+                </div>
+                <Container>
+                    <form>
+                        <FormGroup row>
+                            <Label for="exampleFile" xs={6} sm={4} lg={2} size="lg">Upload</Label>
+                            <Col xs={4} sm={8} lg={10}>
+                                <InputGroup>
+                                    <InputGroup addonType="prepend">
+                                        <Button color="info" style={{ color: "black", zIndex: 0 }} onClick={this.openFileBrowser.bind(this)}><i className="cui-file"></i> Browse&hellip;</Button>
+                                        &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp;
+                                        <input type="file" hidden onChange={this.fileHandler.bind(this)} ref={this.fileInput} onClick={(event) => { event.target.value = null }} style={{ marginLeft: 500 }} />
+
+                                        <Button className="primary jumbotron-button" onClick={this.onCreateTeamsMeetingsButtonClick.bind(this)} style={{ marginLeft: 250 }}>Create meetings</Button>
```

Comment:
Change the function name here as well and at all references

Reply:
Resolved